bundles/org.openhab.binding.denonmarantz/README.md

```diff
@@ -12,6 +12,9 @@ Tested models: Marantz SR5008, Denon AVR-X2000 / X3000 / X1200W / X2100W / X2200
 Denon models with HEOS support (`AVR-X..00H`) do not support the HTTP API. They do support Telnet.
 During Discovery this is auto-detected and configured.
 
+For devices without ethernet interface you can use a serial-to-TCP-Bridge (e.g. ESP8266 with [Tasmota and TCP Serial Bridge](https://tasmota.github.io/docs/Serial-to-TCP-Bridge/)).
```

Comment:
```suggestion
For devices without an ethernet interface, you can use a serial-to-TCP-Bridge (e.g. ESP8266 with [Tasmota and TCP Serial Bridge](https://tasmota.github.io/docs/Serial-to-TCP-Bridge/)).
```

Reply:
@fwolter : do you know how you or me can validate @Skinah  suggestion ? Once applied, this PR can then be merged.

Reply:
I think this can only be done by the maintainer who proposed the changes. If you propose the changes, you should be able to commit your suggestion afterwards.

Reply:
I think my assumption is wrong. It seems @NightDragon1 has disabled that maintainers may add changes to his/her PR's branch. The easiest solution would be that @NightDragon1 accept the changes @Skinah proposed.

Reply:
Nope, worser. The branch doesn't exist anymore. I did some "optimistic" cleanup and so it's not available anymore.

Reply:
Okay, just tried to restore it... not very successful for far.

Reply:
Can you create a new PR and close this one?